### ISSUE TITLE
[Bug]: AI response text not showing for OpenAI model in LibreChat HuggingFace Demo

### ISSUE THREAD

**Issue by terryds** [danny-avila/LibreChat#3965]
### What happened?

Hi, in the huggingface demo, there's something wrong with the response text for OpenAI models.
The response text is not showing at all.
![image](https://github.com/user-attachments/assets/6dbca547-3bfd-4cec-882f-43ebc67189e3)

I tried to look at my browser console and it seems that the response might have been successful since the response object contains valid value for every other properties (there's promptTokens count), but the text is just an empty string.

```
{
    "final": true,
    "conversation": {
        "_id": "66df806946ace904151fa014",
        "conversationId": "cbdb5c3b-6e4d-4687-a309-63705fc11be4",
        "user": "66beb2d49ec7792ee0241dec",
        "__v": 0,
        "createdAt": "2024-09-09T23:10:33.405Z",
        "endpoint": "openAI",
        "isArchived": false,
        "messages": [
            "66df806946ace904151f9fa1",
            "66df806946ace904151fa02a",
            "66df806e46ace904151fa305",
            "66df806e46ace904151fa359"
        ],
        "model": "gpt-4o",
        "resendFiles": true,
        "tags": [],
        "title": "New Chat",
        "updatedAt": "2024-09-09T23:10:38.491Z"
    },
    "title": "New Chat",
    "requestMessage": {
        "messageId": "dc12f6ed-0c66-4b07-a13a-c4ae9a5bd435",
        "parentMessageId": "025d145a-3701-4175-8b0a-317789b9f344",
        "conversationId": "cbdb5c3b-6e4d-4687-a309-63705fc11be4",
        "sender": "User",
        "text": "what",
        "isCreatedByUser": true,
        "tokenCount": 5
    },
    "responseMessage": {
        "messageId": "200227f2-a2b9-4532-bf66-81c3740b53fc",
        "conversationId": "cbdb5c3b-6e4d-4687-a309-63705fc11be4",
        "parentMessageId": "dc12f6ed-0c66-4b07-a13a-c4ae9a5bd435",
        "isCreatedByUser": false,
        "model": "gpt-4o",
        "sender": "GPT-4",
        "promptTokens": 17,
        "endpoint": "openAI",
        "text": ""
    }
}
```

![image](https://github.com/user-attachments/assets/c474ec4b-6c69-43e3-90b0-72051c199f5e)

### Steps to Reproduce

1. Go to the official huggingface demo: https://librechat-librechat.hf.space/
2. Choose OpenAI as the provider
3. Insert API Key
4. Try chatting and it will always give empty text

### What browsers are you seeing the problem on?

Chrome

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by danny-avila** [comment]
I'm not sure why this is happening to you, it could be an issue with the API, but I can't recreate it using the LibreChat HuggingFace Demo or otherwise.

Recorded this using the demo:
https://github.com/user-attachments/assets/50560b73-4a3f-4d25-8778-ffb2f80da660

**Comment by danny-avila** [comment]
I will take a look at the database for the demo, since I can see the exact message ID you experienced this issue with.

**Comment by terryds** [comment]
@danny-avila 

I've finally figured what the issue was.

The issue is from the API Key not having access to the GPT 4 model.
I figured this out by trying chatbotui.com with the same key, and they display the error message "GPT-4 model doesn't exist or you don't have access to it"

I've solved the issue by using API key from my other organization account instead which has access to GPT-4.

I will create a new issue about this UI improvement to display relevant error message
